src/Nether.Analytics.DataLake/Nether.Analytics.DataLake.csproj

```diff
@@ -19,10 +19,14 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.DataLake.Analytics" Version="2.2.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.DataLake.Store" Version="2.1.3-preview" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="1.1.2" />
```

Comment:
I don't think we need to use configuration inside this project, since all necessary configuration parameters should be passed in with constructors of other properties.